docker-compose.yml

```diff
@@ -149,7 +150,6 @@ services:
             - type: volume
               source: orthanc-anon-data
               target: /var/lib/orthanc/db
-            - ${PWD}/orthanc/orthanc-anon/config:/run/secrets:ro
```

Comment:
I'm assuming this is because now we're manually copying and modifying the `orthanc.json` in the Dockerfile, but double-checking that this is the case and this was not accidental.

Reply:
yep that's right